wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -1,27 +1,81 @@
 {% load markdown_deux_tags %}
+{% load humanize %}
+{% load static %}
+
 {% if object.metadata.cancelled_election.title %}
     <h2>{{ object.metadata.cancelled_election.title }}</h2>
 {% else %}
-    <h2>Cancelled Election</h2>
+    <div>
```

Comment:
Just out of interest, why the empty `div` here? Is it to do with spacing / stacking?

Reply:
It's not doing anything here, will remove.